lib/utils/get-note.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import filterNotes from './filter-notes';
+
+export default function getNote( state ) {
+	if ( state.note ) {
+		return state.note;
+	}
+	const filteredNotes = filterNotes( state );
+	const noteIndex = Math.max( state.previousIndex, 0 );
+	return filteredNotes[ noteIndex ];
+}
```

Comment:
seems like this could have a better name. is it getting _the current_ note or the _selected_ note or the _active_ note?